Exiled.Events/EventArgs/EnteringPocketDimensionEventArgs.cs

```diff
@@ -36,6 +38,11 @@ public EnteringPocketDimensionEventArgs(Player player, Vector3 position, bool is
         /// </summary>
         public Player Player { get; }
 
+        /// <summary>
+        /// Gets the Scp106 who sent the player to the pocket dimension.
```

Comment:
the majority of API have the scps formatted as "SCP-XXX" in the documentation (not ScpXXX)